app/views/shared/_dap_analytics.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= t('notices.dap_html') %>
+<% dap_source = 'https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA' %>
+<%= nonced_javascript_tag({src: dap_source, async: true, id: '_fed_an_ua_tag'}) do %>
```

Comment:
Weirdly enough, it does. Although the method signature contains a parameter called 'content_or_options', it will always try and put the first the argument into the tag as content, unless you also supply a block. See the code [here](https://github.com/twitter/secureheaders/blob/master/lib/secure_headers/view_helper.rb#L106).

I verified this by just passing hash to `nonced_javascript_tag`, the method shown above tried to call `html_safe` on it. Passing a string 'works', although it's put inside the tag as content.